iron-form.html

```diff
@@ -451,7 +451,9 @@
           if (this._usesCheckedInsteadOfValue(el)) {
             el.checked = this._customElementsInitialValues[i];
           } else {
-            el.value = this._customElementsInitialValues[i];
+            // The native input/textarea displays literal "undefined" when its
+            // its value is set to undefined, so default to null instead.
+            el.value = this._customElementsInitialValues[i] || null;
```

Comment:
This has the effect of turning `''` into `null` but the comment only talks about `undefined`. This broke a downstream client when they updated.

Should we be more explicit here, with something like:

```javascript
var value = el.value = this._customElementsInitialValues[i];
if (value === undefined) {
  value = null;
}
el.value = value;
```